### ISSUE TITLE
Disabling negative voting result causes lack of signs

### ISSUE THREAD

**Issue by umputun** [umputun/remark42#727]
Enabling `POSITIVE_SCORE` changes existing votes as well. This is expected (I guess) but confusing side-effect.

<img width="180" alt="nbqwyv3bqe-20200607-131121" src="https://user-images.githubusercontent.com/535880/83976581-6e8ad180-a8c0-11ea-9012-c00d8f963daf.png">

Not sure how to deal with this, but can think of 3 possible solutions:

1. do nothing
1. UI not just loose +/- on POSITIVE_SCORE, but also shows all negative values as 0s
1. UI checks if any vote for the post negative and shows +/- in this case (POSITIVE_SCORE mode only)

/cc @paskal @akellbl4 @Mavrin

**Comment by akellbl4** [comment]
In my opinion only positive scores should have different view with like button or something like that. Downvote button in this situation absolutely useless.

The second thing if we show downvoted comments as zero-voted it will be strange just because upvoting will do nothing if comment is downvoted. 
Please correct me if I’m wrong.

**Comment by umputun** [comment]
> Downvote button in this situation absolutely useless.

No, it is not. Positive-only score allow both positive and negative voting it just limits "negative" value to 0

> The second thing if we show downvoted comments as zero-voted it will be strange just because upvoting will do nothing if comment is downvoted.

good point.

**Comment by akellbl4** [comment]
Okay, I didn’t catch idea from the first time, now I understand.
I thought it means you could vote only positive.

I think current behavior is predictable for current situation, the only thing I could make is to make downvote button a bit transparent (disabled state), and it will show that this button isn’t clickable without hovering.

**Comment by umputun** [comment]
the issue I'm having is not a behavior but how it shows the existing negative scores. Both negative and positive are very similar due to the lack of sign and the color-blinded users won't even notice the difference. If conditional signs (3) not that easy to implement maybe we should show a negative score with some other style change, like italic or smth like this.

**Comment by esvyridov** [comment]
I think that another possible solution can be rendering negative score as it is (with "-" sign), but the downvote button must be disabled. In other words, transition from POSITIVE_SCORE from `false` to `true` will not have any effect on score of existing comments, even if they were downvoted, but these comments can be upvoted only. Score behaviour of new comments won't be changed (disabled downvote button).

**Comment by paskal** [comment]
The [vote component rewrite](https://github.com/umputun/remark42/commit/0e4ae6e050c5312625c5aa1d7693043a64f0dd62), released in [v1.10.0](https://github.com/umputun/remark42/releases/tag/v1.10.0), renders the signed score directly. Existing negative scores retain their minus sign when `POSITIVE_SCORE` is enabled, so they remain distinguishable without relying on colour.
